app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, t(".title")) %>
```

Comment:
1. lượt bỏ dấu ngoặc của `provide`
2. Sử dụng I18n cho toàn bộ file html lun, chứ ko phải sử dụng tiêu biểu như thế này nhé